README.rst

```diff
@@ -119,7 +119,7 @@ These are both equivalent to running::
 
 If several config files are present, they are read in the following order:
 
-#. ``pyproject.toml``
+#. ``pyproject.toml`` (only if the ``tomli`` library is available)
```

Comment:
or Python >= 3.11.

Reply:
Is there something I am missing? This wouldn't work: Python 3.11 ships the tomllib library, not tomli. Yet codespell only has code for tomli.

For Python 3.11 to "work out of the box", someone would have to make a PR that introduces a logic like:
- if Python version >= 3.11, use tomllib
- Else check if tomli installed, if yes: use it, if no: raise (or not)

or am I wrong?

see also what I wrote in https://github.com/codespell-project/codespell/pull/2552#discussion_r1003224716

Reply:
I was going from https://github.com/codespell-project/codespell/blob/master/pyproject.toml#L38 ... now I understand and will create a PR.